networking/multiple_networks/primary_networks/about-user-defined-networks.adoc

```diff
@@ -15,13 +15,17 @@ UDN improves the flexibility and segmentation capabilities of the default Layer
 
 The following diagram shows four cluster namespaces, where each namespace has a single assigned UDN, and each UDN has an assigned custom subnet for its pod IP allocations. The OVN-Kubernetes handles any overlapping UDN subnets. Without using the Kubernetes network policy, a pod attached to a UDN can communicate with other pods in that UDN. By default, these pods are isolated from communicating with pods that exist in other UDNs. For microsegmentation, you can apply the Kubernetes network policy within a UDN. You can assign one or more UDNs to a namespace, with a limitation of only one primary UDN to a namespace, and one or more namespaces to a UDN.
 
-image::527-OpenShift-UDN-isolation-012025.png[Namespace isolation concept in a user-defined network (UDN)]
+image::527-OpenShift-UDN-isolation-012025.png[The namespace isolation concept in a user-defined network (UDN)]
 
 [NOTE]
 ====
 Support for the Localnet topology on both primary and secondary networks will be added in a future version of {product-title}.
 ====
 
+The following diagram shows tenant isolation where each tenant has a single assigned UDN. A tenent acts as a disconnected network that is isolated from other tenants. For a namespace that exists in a tenant, pods in that namespace can use the UDN to communicate with pods from another namespace that exists in the same tenant.
```

Comment:
nit: the diff between namespace isolation and tenant isolation as shown is really just to iterate more than 1 namespace can be part of same tenant.. so this usecase one can use CUDN , 1st use case one can use UDN.

Reply:
Thanks, updated to state:

> The following diagram shows tenant isolation where each tenant has a single assigned UDN and uses the `ClusterUserDefinedNetwork` CR to define the UDN across multiple namespaces.